roles/openshift_storage_glusterfs/templates/v3.6/glusterfs-storageclass.yml.j2

```diff
@@ -3,10 +3,6 @@ apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: glusterfs-{{ glusterfs_name }}
-{% if glusterfs_storageclass_default is defined and glusterfs_storageclass_default %}
-  annotations:
-    storageclass.kubernetes.io/is-default-class: "true"
-{% endif %}
 provisioner: kubernetes.io/glusterfs
```

Comment:
Removing this chunk is a bug, but not one introduced by the backport. I'm going to allow this to go forward, submit a PR to fix it against master, and we'll backport it in a different PR. :)

Reply:
cool :) will do

Reply:
@jarrpa what do you want to do with this? this is 3.6 file in 3.7 branch. I made sure this is didn't got into 3.6 branch. And nobody should use 3.6 code in 3.7 branches, so its kinda absolute. Do you want to align them or just leave it as it is.

Reply:
Eh, leave it for now. It'll all get deleted when the version collapse PR gets backported.